### ISSUE TITLE
After enabling add on, the Frame rate will be locked (Blender4.1.1)

### ISSUE THREAD

**Issue by Yorha4D** [pKrime/Expy-Kit#52]
After enabling add on, the Frame rate will be locked (Blender4.1.1)
![image](https://github.com/pKrime/Expy-Kit/assets/116185401/5342c18e-bf52-4c1e-a8fe-059793eaa0c2)
@pKrime

**Comment by MikeX7** [comment]
Yeah i have the same issue, seems like the addons breaks the blender preset manager so doing things like changing fps/resolution/cloth sim presets etc is not possible while the addon is enabled.
